tests/bootstrap_redis.sh

```
@@ -0,0 +1,110 @@
+
+add_to_hosts() {
+    local hostname=${1}
+    local container_ip=${2}
+    if [ -n "${GITHUB_ENV}" ]; then
+         sudo sh -c "echo -ne '\n${container_ip} ${hostname}' >> /etc/hosts"
+    fi
+}
+
+cli() {
+  docker run -v "${SSL_PATH}:/ssl-certs" --rm "${NODE_IMAGE}" \
+    "${CLI_COMMAND}" ${client_args[@]} "$@"
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)